tcpkeepalive_darwin.go

```diff
@@ -0,0 +1,34 @@
+package tcpkeepalive
+
+import (
+	"os"
+	"runtime"
+	"syscall"
+	"time"
+)
+
+const sysTCP_KEEPINTVL = 0x101
+
+func setIdle(fd uintptr, d time.Duration) error {
+	// not possible with darwin
+	return nil
+}
+
+func setCount(fd uintptr, n int) error {
+	// not possible with darwin
```

Comment:
Looking at the source code https://github.com/apple/darwin-xnu/blob/0a798f6738bc1db01281fc08ae024145e84df927/bsd/netinet/tcp.h looks like also count is implemented on darwin (not sure from which version)

In this repo that uses the hackish way of calling setsockopt they aren using it:
https://github.com/felixge/tcpkeepalive/blob/master/keepalive_darwin.go